Makefile

```diff
@@ -2,15 +2,15 @@
 #   rsync
 #   wget or curl
 #
-JRUBY_VERSION=1.7.11
-ELASTICSEARCH_VERSION=1.1.1
+JRUBY_VERSION=1.7.17
```

Comment:
.18 seem fine also, I don't see any security problems with this version looking at the security pages or the changelog.